histomics_label/web_client/package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "@girder/wsi_superpixel_guided_labeling",
+    "name": "@girder/histomics_label",
     "version": "0.1.0",
     "private": true,
     "description": "Perform active learning with superpixel classification.",
```

Comment:
Update the home page link in anticipation of the change (and the one in setup.py, too).

Reply:
Fixed, thanks!